### PR TITLE
rendering_graph_node layer table updated with gcc extensions for legibility

### PR DESCRIPTION
**Pull request by someone2639**
More things in the repo should do this

### CODE REVIEW

File: src/game/rendering_graph_node.c [hunk 1]
```diff
@@ -198,41 +194,114 @@ static const Gfx dl_silhouette_end[] = {
 struct RenderPhase {
     u8 startLayer;
     u8 endLayer;
-#ifdef OBJECTS_REJ
+// #ifdef OBJECTS_REJ
```
Comment (someone2639) [inline src/game/rendering_graph_node.c:197]: a side effect of naming struct fields explicitly is that this field has to stay; opinions welcome

Comment (someone2639) [reply src/game/rendering_graph_node.c]: Update: this is wrong and no longer needs opinions

**Comment by gheskett** [comment]
I've honestly got no idea what's going on here lol, I might abstain from reviewing this one.

**Review by arthurtilly — APPROVED** [review]
approved

**Comment by Arceveti** [comment]
lgtm if there are no merge conflicts

**Review by thecozies — APPROVED** [review]
it would save my life

**Review by gheskett — APPROVED** [review]

**Pull request merged**